include/seqan/journaled_set/score_biaffine.h

```diff
@@ -90,13 +90,6 @@ class Score<TScoreValue, BiAffine>
                                                                         _gapOpenHorizontal(gapOpenHorizontal),
                                                                         _gapExtendVertical(gapExtendVertical),
                                                                         _gapOpenVertical(gapOpenVertical) {}
-
-    Score(Score const & other) : _match(other._match),
-                                 _mismatch(other._mismatch),
-                                 _gapExtendHorizontal(other._gapExtendHorizontal),
-                                 _gapOpenHorizontal(other._gapOpenHorizontal),
-                                 _gapExtendVertical(other._gapExtendVertical),
-                                 _gapOpenVertical(other._gapOpenVertical) {}
 };
```

Comment:
No, the copy constructor will be defaulted since the custom constructs won't affect this.
Otherwise rule-of-5- applies 😫. Probably the best would be aggregate type, but then it is not consistent with other score types and I don't see the point of changing all this for this small class, which is used in only one place at the moment. The other constructors are for consistency with the Simple score class.